omod/src/test/java/org/openmrs/module/idgen/web/controller/IdentifierSourceRestControllerTest.java

```diff
@@ -141,8 +140,8 @@ public void shouldUploadIdentifiersFromSource() throws Exception {
         SimpleObject identifiersAfterUpload = deserialize(handle(getRequestAfterUpload));
         assertThat(
                 identifiersAfterUpload.get("identifiers").toString(), allOf(
-                containsString("Test Identifier Pool: G-0"),
```

Comment:
Why did you change these?

Reply:
It's because of this change (https://github.com/openmrs/openmrs-module-idgen/pull/74/files#diff-37ab2fee2e4e68dcf200614c09c4a51dR24 -PooledIdentifier). I removed the the IdentifierPool from PooledIdentifier model since its a cyclic dependency and could cause serialization issues with Jackson.